app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/utils/rules.ts

```diff
@@ -96,3 +102,47 @@ export const checkIsRuleGroupDisabled = (allRecordsMap: Record<string, StorageRe
     return true;
   } else return false;
 };
+
+export const getSampleRules = async () => {
+  try {
+    const templates = await getTemplates();
```

Comment:
Why are we filtering here after fetching all templated. We should fetch only sample templates by id `getTemplates([1, 2, 3, 4, 5])`